### ISSUE TITLE
do demo with constant-PSF imaging

### ISSUE THREAD

**Issue by davidwhogg** [davidwhogg/TheThresher#69]
yes, constant PSF!  This proves that at constant PSF, The Thresher returns the image stack.

**Comment by drphilmarshall** [comment]
Well - not your grandmother's stack! Won't the result will be a set of
well-sampled, lightly deconvolved images (hopefully with no Gibbs
oscillations) with _common resolution_, that can be sensibly combined
in some way?

On Tue, Jun 26, 2012 at 5:57 PM, David W. Hogg
reply@reply.github.com
wrote:

**Comment by davidwhogg** [comment]
No, if there is no variation in PSF, there is literally _no_ deconvolution applied _at all_.  I think!

**Comment by drphilmarshall** [comment]
Ohhh because the Thresher won't have anything to tell it what is an
extended object and what is a point source?
I think the regularisation will be the tie-breaker then - some schemes
will favour darker scenes and bigger PSFs, and hence more point-like
objects...
If the regularisation is the same for the PSF and the scene, maybe
it'll return the (denoised) data?
Maybe this test is a good one for deciding what regularisation you
want to use? It's a good thing you all at Camp Hogg love assigning
priors.

On Tue, Jun 26, 2012 at 6:27 PM, David W. Hogg
reply@reply.github.com
wrote:
